holoviews/operation/datashader.py

```diff
@@ -135,7 +135,7 @@ def _get_sampling(self, element, x, y):
 
         xtype = 'numeric'
         if isinstance(xstart, datetime_types) or isinstance(xend, datetime_types):
-            xstart, xend = dt_to_int(xstart), dt_to_int(xend)
+            xstart, xend = dt_to_int(xstart, 'ns'), dt_to_int(xend, 'ns')
```

Comment:
Wondering if life might be easier/ less error-prone if instead of ``dt_to_int(x, 'ns')`` and ``dt_to_int(x)`` you had two separate utilities: ``dt_to_ns`` and ``dt_to_us``...

Reply:
Hmm, not sure how that's either easier or less error-prone. The utility is general purpose and in almost all cases the default of 'us' is appropriate.

Reply:
A single utility seems appropriate, as physicists might someday want time resolutions other than ns and us...

Reply:
It just seems to be a pain to specify 'ns' everywhere. Anyway, it was just a suggestion and I don't really know how to make things easier...